packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -44,6 +51,13 @@ export class LogContext {
   private readonly launchArgs: LaunchRequestArguments;
   private readonly logLines: string[] = [];
   private state: DebugLogState | undefined;
+  private frameHandles = new Handles<ApexDebugStackFrameInfo>();
```

Comment:
Each of the handles are a map from a unique ID (needed for when you expand elements) to an object. This is needed for the UI to correctly retrieve information about whatever frame, scope, or variable is clicked on.